src/main/java/tickbot/Tickbot.java

```diff
@@ -0,0 +1,25 @@
+package tickbot;
+
+import java.util.Objects;
+
+import tickbot.ui.Ui;
+import tickbot.ui.graphics.GraphicsUi;
+import tickbot.ui.text.TextUi;
+
+/**
+ * The main class of the application.
+ */
+public class Tickbot {
+    /**
+     * The main entrance of Tockbot.
```

Comment:
Is this a typo? `Tockbot` instead of `Tickbot`

Reply:
Fixed. Thanks!